src/api/app/assets/stylesheets/webui/in_place_editing.css

```diff
@@ -0,0 +1,3 @@
+.in-place-editing .hidden {
```

Comment:
We can avoid having this extra rule by using the Bootstrap class `d-none`, which does the same.

Reply:
True